openedx/core/djangoapps/oauth_dispatch/toggles.py

```diff
@@ -0,0 +1,21 @@
+"""
+Toggles for Oauth Dispatch.
+"""
+
+
+from edx_toggles.toggles import WaffleSwitch
+
+OAUTH_DISPATCH_WAFFLE_SWITCH_NAMESPACE = 'oauth_dispatch'
+
+# .. toggle_name: DISABLE_JWT_FOR_MOBILE
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Toggle to control creation of JWT token for mobile
```

Comment:
what about something like:

Set toggle to True to ignore mobile requests for JWTs, and instead provide the legacy opaque Bearer token. This toggle can be used as a temporary kill switch as the mobile app attempts to rollout the transition to JWT authentication for the first time.